Cannot compile plugin on Mac OSX

Hello, I am currently unable to compile this plugin on Mac OSX, when compiling, this is the error message getting sent out.
```

Link UE4Editor-GenericGraphEditor.dylib

Undefined symbols for architecture x86_64:

"EKeys::Invalid", referenced from:
FEditorCommands_GenericGraph::RegisterCommands() in Module.GenericGraphEditor.cpp.o

ld: symbol(s) not found for architecture x86_64

clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

--- a/Source/GenericGraphEditor/GenericGraphEditor.Build.cs
+++ b/Source/GenericGraphEditor/GenericGraphEditor.Build.cs
@@ -39,6 +39,7 @@ public class GenericGraphEditor : ModuleRules
                 "GenericGraphRuntime",
                 "AssetTools",
                 "Slate",
+                "InputCore",
                 "SlateCore",
                 "GraphEditor",
                 "PropertyEditor",

